SUMMARY.md

```diff
@@ -25,3 +25,4 @@
 
 * [Extras](extras/README.md)
   * [Adding your static page to Django](extras/adding-your-static-page-to-django.md)
+  * [Django Girls tutorial extensions](https://tutorial.djangogirls.org/en/django/)
```

Comment:
That doesn't link to extensions, but just to the main Django page.

Reply:
True! Fixed in a new commit :)